core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpSamDAO.scala

```diff
@@ -37,6 +37,17 @@ class HttpSamDAO(baseSamServiceURL: String)(implicit val system: ActorSystem) ex
     }
   }
 
+
+  override def getUserStatus(userInfo: UserInfo): Future[Option[UserStatus]] = {
+    val url = samServiceURL + "/register/user"
+    retry(when500) { () =>
+      pipeline[Option[UserStatus]](userInfo) apply Get(url) recover {
+        case notOK: UnsuccessfulResponseException if StatusCodes.Conflict == notOK.response.status => None
```

Comment:
should this handle Not_Found instead of Conflict? I don't think /register/user ever returns conflict